src/v2/modules/documents/routes.js

```diff
@@ -33,6 +33,27 @@ exports.getDocuments = {
   }
 };
 
+exports.postDocument = {
+  method: 'POST',
+  path: '/crm/2.0/documents',
+  handler: (request, h) => entityHandlers.createEntity(request, h, 'document'),
+  options: {
+    description: 'Add a document for a licence number',
+    validate: {
+      payload: {
+        regime: Joi.string().required().valid('water'),
```

Comment:
I don't think this conforms to the pattern Mark set up in some of the other routes.
For example, for regime, the route handler would check the basic shape/type of data:
```
regime : Joi.string().required()
```
A validation error here would give a 400 bad request.

Whereas at the service level, an additional more strict validation would occur:
```
regime : validators.REGIME
```
A validation error here would be mapped to a 422 unprocessable entity response

Reply:
done - makes sense. I was wondering why we validate the data in two places - felt like duplication.